sycl/include/sycl/handler.hpp

```diff
@@ -559,7 +559,8 @@ class __SYCL_EXPORT handler {
 
   /// Saves the location of user's code passed in \p CodeLoc for future usage in
   /// finalize() method.
-  /// TODO: remove the first version of this func (the one without the IsTopCodeLoc arg)
+  /// TODO: remove the first version of this func (the one without the
+  /// IsTopCodeLoc arg)
   ///   at the next ABI breaking window since removing it breaks ABI on windows.
   void saveCodeLoc(detail::code_location CodeLoc);
```

Comment:
Instead of a TODO, how about we remove `saveCodeLoc(detail::code_location CodeLoc)` right away, under preview breaking flag?
Like:
```
ifndef __INTEL_PREVIEW_BREAKING_CHANGES
  void saveCodeLoc(detail::code_location CodeLoc);
#endif
```

Reply:
Apologies this is just an unrelated formatting change to the comment that snuck in and is not related to my PR at all. I've removed it now.

Reply:
Cool. I've made the PR to remove the TODO: https://github.com/intel/llvm/pull/19276